### PR TITLE
Return Klaytn network type with getNetworkType function

### PR DESCRIPTION
**Pull request by jimni1222**
## Proposed changes

Modify getNetworkType function to return Klaytn network type.

## Types of changes

Please put an x in the boxes related to your change.

- [x] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-js/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-js)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

close https://github.com/klaytn/caver-js/issues/16

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: test/getNetworkType.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/*
+    Copyright 2018 The caver-js Authors
```
Comment (kjhman21) [inline test/getNetworkType.js]:
```suggestion
    Copyright 2019 The caver-js Authors
```

Comment (kjhman21) [reply test/getNetworkType.js]: Please make sure new license statements should include the corresponding year.

Comment (jimni1222) [reply test/getNetworkType.js]: Oh, okay my bad ;-;! Thank you for reviewing

**Comment by jimni1222** [comment]
This PR will be merged after updating with the hash value of the Genesis changed after the baobab reset. (Genesis block change is under discussion.)

**Pull request merged**